llvm/test/CodeGen/X86/atomicvec-float.ll

```diff
@@ -0,0 +1,38 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc < %s --mtriple=x86_64 | FileCheck %s
```

Comment:
There is already a deceptively named atomic-fp.ll. llvm/test/CodeGen/X86/atomic-non-integer.ll seems like it already exists?